### ISSUE TITLE
Uncaught TypeError: Cannot read property 'prototype' of undefined

### ISSUE THREAD

**Issue by JoeNoPhoto** [Nicolab/atom-local-history#30]
[Enter steps to reproduce below:]

1. Just saving a simple file. First error happened with a .rb file, this tract is form a .scss save

**Atom Version**: 0.205.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: Mac OS X 10.10.3
**Thrown From**: [local-history](https://github.com/Nicolab/atom-local-history) package, v3.1.0

### Stack Trace

Uncaught TypeError: Cannot read property 'prototype' of undefined

```
At /Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:759

TypeError: Cannot read property 'prototype' of undefined
  at Object.module.exports.extends (/Users/.atom/packages/local-history/node_modules/atom-helpers/lib/utils.js:25:26)
  at Object.<anonymous> (/Users//.atom/packages/local-history/lib/local-history-view.js:28:16)
  at Module._compile (module.js:452:26)
  at Object.loadFile [as .js] (/Applications/Atom.app/Contents/Resources/app.asar/src/babel.js:162:21)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.module.exports.getView (/Users//.atom/packages/local-history/lib/local-history.js:82:31)
  at /Users//.atom/packages/local-history/lib/local-history.js:42:17
  at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at TextBuffer.module.exports.TextBuffer.saveAs (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1021:20)
  at TextBuffer.module.exports.TextBuffer.save (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:1014:19)
  at TextEditor.module.exports.TextEditor.save (/Applications/Atom.app/Contents/Resources/app.asar/src/text-editor.js:578:26)
  at Pane.module.exports.Pane.saveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:523:18)
  at Pane.module.exports.Pane.saveActiveItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:506:19)
  at Workspace.module.exports.Workspace.saveActivePaneItem (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace.js:585:35)
  at atom-workspace.atom.commands.add.core:save (/Applications/Atom.app/Contents/Resources/app.asar/src/workspace-element.js:314:30)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -2:14.3.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:14.3.0 editor:newline (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:14.1.0 emmet:insert-formatted-line-break-only (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:14.1.0 editor:newline (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:13.5.0 core:move-up (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:12.7.0 core:paste (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:08.5.0 editor:consolidate-selections (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:08.5.0 core:cancel (atom-text-editor.editor.cursor-i-beam.is-focused)
     -2:06.2.0 core:backspace (atom-text-editor.editor.cursor-i-beam.is-focused)
     -1:51 core:save (atom-text-editor.editor.cursor-i-beam.is-focused)
     -1:14.5.0 deprecation-cop:view (atom-workspace.workspace.scrollbars-visible-always.cursor-i-beam.theme-base16-tomorrow-dark-theme.theme-one-dark-ui)
     -0:44.8.0 tree-view:remove (ol.tree-view.list-tree.has-collapsable-children.focusable-panel.multi-select)
     -0:22.1.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:09.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:04.1.0 core:paste (atom-text-editor.editor.cursor-i-beam.is-focused)
     -0:02.2.0 core:save (atom-text-editor.editor.cursor-i-beam.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "vim-mode",
      "jquery-snippets"
    ],
    "projectHome": "/Users//Projects",
    "themes": [
      "one-dark-ui",
      "base16-tomorrow-dark-theme"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.27.10
atom-html-preview, v0.1.6
block-cursor, v0.12.5
color-picker, v2.0.4
emmet, v2.3.10
file-icons, v1.5.7
flex-tool-bar, v0.3.0
git-log, v0.4.1
javascript-snippets, v1.0.0
language-babel, v0.6.2
language-javascript-jsx, v0.3.6
linter, v0.12.6
linter-csslint, v0.0.13
linter-eslint, v0.5.8
linter-htmlhint, v0.0.16
linter-scss-lint, v1.0.4
linter-tidy, v1.0.1
local-history, v3.1.0
minimap, v4.9.2
minimap-autohide, v0.10.0
minimap-bookmarks, v0.1.0
minimap-color-highlight, v4.1.4
minimap-find-and-replace, v4.2.0
minimap-git-diff, v4.1.3
minimap-highlight-selected, v4.3.0
minimap-selection, v4.2.0
project-manager, v1.15.7
script, v2.23.0
tool-bar, v0.1.5

# Dev
No dev packages
```

**Comment by Nicolab** [comment]
Thank you, I will look. It's related with #27, #28 and #29.

**Comment by Nicolab** [comment]
Fixed by #df14c2ce3 and released to v3.2.0

Thanks!
